packages/rum_sdk/example/lib/main.dart

```diff
@@ -73,7 +75,7 @@ class HomePage extends StatefulWidget {
   const HomePage({Key? key}) : super(key: key);
 
   @override
-  _HomePageState createState() => _HomePageState();
+  State<HomePage> createState() => _HomePageState();
```

Comment:
Since the `_HomePageState` is private, it is a better approach to not expose it and define this as `State<HomePage>`